server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultConnectOptionsContext.java

```diff
@@ -56,11 +56,12 @@
 
     private static final long DEFAULT_WS_INACTIVITY_TIMEOUT_MILLIS = 0L;
     private static final int DEFAULT_HTTP_KEEPALIVE_TIMEOUT = 30; //seconds
+    private static final int DEFAULT_HTTP_KEEPALIVE_MAX_CONNECTIONS = 5;
```

Comment:
See feedback for HttpResourceAddress about having only one constant that is shared to avoid cases where default values diverge.